doc/src/devdocs/compiler.md

```diff
@@ -18,7 +18,7 @@ ensuring they are valid both for the current runtime and after deserialization.
 
 When emitted into the object file, these globals are stored as references
 in a large `gvals` table. This allows the deserializer to reference them by index,
-and implement a custom manual GOT-like mechanism to restore them.
+and implement a custom manual Global Offset Table (GOT) mechanism to restore them.
```

Comment:
GOT means, well, the GOT in a object file and ours is certainly not that and with a very different format.

I mostly mean that this should probably be `GOT-like mechanism` or `GOT` because GOT is a noun.